pkg/installer/arobundsasigningkey.go

```diff
@@ -0,0 +1,69 @@
+package installer
```

Comment:
Potentially small nit: filename has a typo. Likely should be aro*bound*sasigningkey.go?

Reply:
I'll fix this.